sdk/core/azure-core/src/main/java/com/azure/android/core/util/logging/LogLevel.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.core.util.logging;
+
+/**
+ * Enum which represent logging levels used in Azure SDKs.
+ */
+public enum LogLevel {
+    /**
+     * Indicates that log level is at verbose level.
+     */
+    DEBUG(1),
```

Comment:
I vaguly recall once Brandon told about IntDef, so was looking into IntDef and learned that ProGuard by default optimize enum to int, so we should be fine with enum @bsiegel ?

https://www.guardsquare.com/en/products/proguard/manual/usage/optimizations
https://stackoverflow.com/a/32236950

Reply:
Based on the discussion in that SO post, it seems like that optimization may not work, or at least it's very difficult to get ProGuard to enable it. I'd suggest going with IntDef etc. unless we find that to be super burdensome.

Reply:
Good catch Brandon!.  @vcolin7 please see this https://developer.android.com/reference/androidx/annotation/IntDef.html

Reply:
Done :)